sdk/python/packages/flet/src/flet/controls/page.py

```diff
@@ -349,6 +357,11 @@ class Page(BasePage):
     Called when brightness of app host platform has changed.
     """
 
+    on_orientation_change: Optional[EventHandler[OrientationChangeEvent]] = None
```

Comment:
I can't find when/where this event is being triggered.

Reply:
I thought it should be just `on_media_change`, like in example to this PR?

Reply:
Nice catch. 
It's an old API remain. Will be removed.